gnovm/pkg/gnolang/values.go

```diff
@@ -1546,6 +1546,11 @@ func (tv *TypedValue) ComputeMapKey(store Store, omitType bool) MapKey {
 		bz = append(bz, pbz...)
 	case *PointerType:
 		fillValueTV(store, tv)
+		// nil pointer.
+		if tv.V == nil {
+			bz = append(bz, nilStr...)
```

Comment:
Let's use `nilStr` when we have interfaces.

For this case, I'd say the most correct is actually:

```
var x [sizeOfUintPtr]byte
bz = append(bz, x[:]...)
```

So that it's all 0 bytes, as nil pointers are.

Reply:
right done here https://github.com/gnolang/gno/pull/4505/commits/dbee3b3cdb787451ba89bdafbb8926f1e57d6021